packages/api/src/external/commonwell/link/get.ts

```diff
@@ -239,11 +236,7 @@ const findAllPersonsStrongId = async (
   queryMeta: RequestMetadata
 ): Promise<Person[]> => {
   const { log } = out("cw.findAllPersonsStrongId");
-  const strongIds = getPersonalIdentifiersFromPatient(patient);
-  if (!strongIds.length) {
-    return [];
-  }
```

Comment:
I reverted it but here's the idea (will put in a different PR).

For both `searchPersons` and `commonwellPersonLinks` the methods take in arrays, and importantly `searchPersons` only calls commonWell if the passed in array is non-empty. Both of these functions should be able to safely handle empty arrays (they do) and pass back a result based on that. If we want to prevent unnecessary calls to a third party, it should be determined by the method being called (i.e. the empty check should be in `searchPersons`). Other methods should be able to safely call these methods with empty lists.... or the typing should reflect forcing a non-empty list.

Thus, we don't need the empty checks in in my mind.

Reply:
> If we want to prevent unnecessary calls to a third party, it should be determined by the method being called (i.e. the empty check should be in searchPersons)

That makes perfect sense. 👍